### DISCUSSION TITLE
Missing Functionality?

### DISCUSSION THREAD

**Discussion by RoyalGreen** [Koenkk/zigbee2mqtt#22500] [General]
Hi there,

I have updated to the latest version of Z2M 1.37.0-1 and updated the firmware to version v7.4.2.0 on my Sonoff E 3.0 coordinator.

There used to be a slider when naming a device which changed the entity names with HA. I don't see this any more, it was useful when i had to repair devices. 

Has it moved somewhere else?

Thanks in advance

**Comment by mrbrdo** [comment]
I also noticed the lack of this function after the recent update. I hope it's a mistake and will come back?
There seems to be no mention in the CHANGELOG about it being removed.

**Reply by francisp2** [reply]
https://github.com/Koenkk/zigbee2mqtt/issues/22424#issuecomment-2094066889

https://github.com/Koenkk/zigbee2mqtt/issues/22465
